### DISCUSSION TITLE
Difference between the mode and high density region when using the marginal or the full posterior distribution

### DISCUSSION THREAD

**Discussion by PHFHobbelen** [easystats/bayestestR#652] [Q&A]
From googling and searching the internet, I understand that the mode and high density region (HDR) of a parameter may depend on whether you use the marginal or the full posterior distribution for calculating these measures, unless the posterior distributions for each parameter are independent (which means they were estimated using independent datasets and priors were independent, too). I am trying to understand this better and thought it would help to estimate the mode and HDR in both ways. From what I have tried so far, the map_estimate() and hdi() functions use the marginal instead of the full posterior distribution. 

Is there a way to estimate the MAP/HDR for the full posterior distribution using the bayestestR package?

Thank you in advance for any help.

**Comment by PHFHobbelen** [comment]
May be the cwi() function can do this., but it returns "comming soon" as output.

**Comment by bwiernik** [comment]
There isn't a general solution for estimating a mode/MAP/HDR in higher dimensions. This package implements one solution for these statistics in 2 dimensions: https://cran.r-project.org/web//packages/hdrcde/hdrcde.pdf

The behavior of expected values in higher dimensions is better understood. Looking into the James-Stein paradox is a good path to learning more about this behavior https://youtu.be/cUqoHQDinCM?si=VDXIlo8ZsOBn_DxE

Behavior of multivariate quantiles for medians and ETIs is also very tricky and depends on what properties of the unidimensional quantile you want to preserve when defining a multivariate quantile. https://arxiv.org/abs/2401.02499

**Reply by PHFHobbelen** [reply]
Thanks for the quick reply! I will look into these links.
